content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,305 @@
+
+The Group Export operation exports resources required for [United States Core Data for Interoperability V1 (USCDI)](https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi). For a full list of resources see the [US Core Profiles](http://hl7.org/fhir/us/core/STU4/#us-core-profiles). When the _type parameter is used it returns only the FHIR resources specified. The _type parameter should be used to filter the resources required for a specific use case. Limiting the number of requested resources in the _type parameter is recommended as it can decrease response times.
+
+### Registering Client Applications
+
+In order to be granted access to the Group Export operation, a Millennium Bulk SMART application must be registered through Cerner's [code Console](http://fhir.cerner.com/smart/#code-console). See [Authorization](http://fhir.cerner.com/authorization/) for additional details on registering client applications and obtaining access tokens. 
+
+Client application developers will need to work with healthcare organizations to identify the available Groups to be exported. Organizations can manage Groups within Ignite Management Tooling.
+
+_Notes_
+
+* Authentication is required, open endpoints do not support bulk export workflows
+* Cerner supports v1.0.1 of the [FHIR Bulk Data Access (FLATFHIR) Implementation Guide](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/)
+* Bulk data runs against an organization's database and utilizes the same services running other applications, so it is important to consider performance. Bulk data exports large amounts of data for large groups of patients, which takes more time to complete the larger the data set or patient population.
+
+### When to use bulk data
+
+The Bulk Data Export API enables connecting applications to mine large amounts of data; however, its inherent flexibility also presents challenges. When implementing an interface between a Cerner Millennium domain and a bulk data app, it is very important that the queries inherent in the bulk data request capture the data necessary for the app – but little or no more. Furthermore, export files should be deleted as soon as they are no longer needed.
+
+There are several factors that affect the amount of time it takes to complete a Bulk API export:
+
+* The number of patients to whom the request applies (that is, the Group)
+* The number of resource types requested
+* The time range of activity being requested
+* Inherent latency due to network issues and effects from concurrent requests to the system
+
+#### Example use cases
+
+* Creating an internal clinical data warehouse for study purposes
+* Consolidation of claims data for comprehensive view
+* Machine learning applications obtaining training data from EHR
+* Integration of population health system with an EHR system
+* Transferring records between EHRs
```

Comment:
@fpivonka  Isn't this more of a seamless exchange case?

Reply:
@SA030800  yeah, looks like it has been removed from current version i think.

Reply:
Yes, this is no longer in the example cases: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R19-R25